tests/NHSD.BuyingCatalogue.Ordering.Api.Testing.Data/Entities/OrderItemEntity.cs

```diff
@@ -81,5 +81,17 @@ FROM dbo.OrderItem
 
             return await SqlRunner.QueryFirstAsync<OrderItemEntity>(connectionString, sql, new { orderId, catalogueItemId });
         }
+
+        public static async Task<int> GetItemIdsForOrderId(
+            string connectionString,
+            int orderId)
+        {
+            const string sql = @"
+                SELECT COUNT(1)
```

Comment:
I think `COUNT(*)` is preferable here. While it won't make any actual difference to the query, I think it's ever so slightly clearer for those less familiar with SQL.

I believe we have used the `COUNT(*)` form elsewhere in our code so it will keep it consistent too.

Reply:
done